### PR TITLE
Setup project structure

### PR DESCRIPTION
**Pull request by Mati-IR**

### CODE REVIEW

File: src/components/menu/Menu.py
Comment (jacekdebski) [inline src/components/menu/Menu.py:1]: move to the parent directory, it is not part of the components

File: src/components/menu/__init__.py
Comment (jacekdebski) [inline src/components/menu/__init__.py:1]: ditto

File: src/components/run.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+
+def main():
+    print("Banking app")
```
Comment (jacekdebski) [inline src/components/run.py:4]: remember to create a better init message for the app :D

File: src/_run.bat [hunk 2]
```diff
@@ -0,0 +1,3 @@
+@echo off
+
+python components/main.py
```
Comment (rfilipczak) [inline src/_run.bat:3]: no such file

**Review by rfilipczak — CHANGES_REQUESTED** [review]

**Review by rfilipczak — APPROVED** [review]

**Review by jacekdebski — APPROVED** [review]

**Pull request merged**